src/server/common/apis/disinfectant/index.js

```diff
@@ -38,7 +70,19 @@ export const fetchDisinfectants = async (type) => {
     )
 
     items = response.payload.toString()
-    return JSON.parse(items).filteredDisinfectants ?? []
+
+    try {
+      await cache.setex(
+        `api:disinfectants:${type}`,
+        config.get('api.disinfectant.cache.TTL'),
+        items
+      )
+    } catch (e) {
+      logger.error(`Failed to cache disinfectants of type ${type}:`, e)
+    }
+
+    const filteredDisinfectants = JSON.parse(items).filteredDisinfectants ?? []
+    return dedupeFitleredDisinfectants(filteredDisinfectants)
```

Comment:
I believe we can remove these two lines, because they will be hit again at the end of the file.